commands/Management/dbump.js

```diff
@@ -0,0 +1,35 @@
+const { MessageEmbed } = require("discord.js")
+
+async function sendEmbed(client, message, args){
+	const guild = client.guilds.cache.get(message.guild.id);
+	const setting = await guild.settings();
+	const bumpingChannel = setting.bumpingChannel;
+	const bumpEmbed = new MessageEmbed()
+	.setAuthor(
+	  "Friendly Bump Reminder",
+	  `${client.user.displayAvatarURL()}`
+	)
+	.setTitle("Heyo all, It's time to Bump")
+	.setURL("https://discord.me/dashboard")
+	.addField("Discord.me", "https://discord.me/dashboard")
+	.addField("Disboard", "`!d bump`")
+	.addField(
+	  "Top.gg",
+	  "https://top.gg/servers/731520035717251142/vote"
+	)
+	.setColor("#8800FF")
+	.setFooter(
+	  "Thanks to Everyone Bumps the server | Once you have bumped, react with ☑ to the post so we know it's been done.\nThis message is automagically sent 2 hours after each disboard bump!"
+	);
+	try{
+	  guild.channels.cache.get(bumpingChannel).send(bumpEmbed);
+	}catch{
+	  console.log("Failed to send Bump message")
+	}
```

Comment:
```suggestion
	guild.channels.cache.get(bumpingChannel).send(bumpEmbed).catch(e => {
	  console.error("Failed to send bump message:");
	  console.error(e);
      })
```

Do not use a try..catch block for promises.